deploy.yml

```diff
@@ -1,12 +1,13 @@
 ---
-- name: Configure and deploy pythondev apps
-  hosts: sirbot
+
+- name: Configure servers
```

Comment:
I think the point of the new "modeling" is to not have anything about configuring a server. Instead each playbook (one for sirbot, one for the site) declares how the server should look and implicitly it's configured.